docs/revsets.md

```diff
@@ -87,6 +87,10 @@ revsets (expressions) as arguments.
 
 * `descendants(x)`: Same as `x::`.
 
+* `reachable(s, b)`: All commits reachable from `s` without entering `::b`.
+  Equivalent to `(b..s)::` for simple graph structures, but distinct when more
+  complex merge commit graphs are involved.
```

Comment:
Yea that makes a lot more sense in hindsight.  I'm renaming this to `reachable_within()` as well since it's a lot more intuitive with the new parameter meaning.

Reply:
I see two-ish votes for keeping it as 'reachable()' so I'll do that